tests/scripts/all.sh

```diff
@@ -3846,8 +3858,19 @@ component_test_full_block_cipher_psa_dispatch () {
     # Configure
     # ---------
 
-    # Start from the full config
-    helper_libtestdriver1_adjust_config "full"
+    common_block_cipher_dispatch 1
+
+    # Disable cipher's modes and AEADs that, when not accelerated, cause
```

Comment:
Suggestion: does putting this in the common function reduce the number of things we have to ignore in `analyze_outcomes.py`?

Reply:
Yes, indeed, this is something I had locally but I didn't pushed yet ;)